@navikt/core/react/src/aksel-provider/AkselProvider.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { createContext, useContext } from "react";
+
+export interface AkselProviderContextType {}
+
+export const AkselProviderContext = createContext<
+  AkselProviderContextType | undefined
+>(undefined);
+
+export interface AkselProviderProps {
+  children?: React.ReactNode;
+  /** Options */
+  options?: {};
+}
+
+export const AkselProvider = ({ children, options }: AkselProviderProps) => {
+  const ctx = useContext(AkselProviderContext);
+
+  if (ctx) {
+    console.warn("AkselProvider should not be nested.");
```

Comment:
Hvorfor er det et problem?

Reply:
Kan i fremtiden ende opp med å sette globale settings her, eks som [her](https://github.com/Shopify/polaris/blob/main/polaris-react/src/components/AppProvider/AppProvider.tsx). kan være litt uheldig å neste den da. Men kan også utvikles videre med tanke på at den kan nestes da.

Reply:
Jeg ser Spectrum in Provider er ment å kunne nestes for å f.eks. sette disabled på en gruppe skjemaelementer: https://react-spectrum.adobe.com/react-spectrum/Provider.html#property-groups